app/src/main/res/layout/activity_main.xml

```diff
@@ -20,35 +20,49 @@
             android:layout_height="wrap_content"
             android:layout_marginStart="20dp"
             android:contentDescription="@null"
+            android:visibility="gone"
             app:srcCompat="@drawable/feed_header_wordmark"
             app:tint="?attr/primary_color" />
 
-        <com.google.android.flexbox.FlexboxLayout
-            android:id="@+id/flexboxLayout"
+        <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            app:flexWrap="wrap"
-            app:justifyContent="flex_start"
-            app:alignItems="center">
+            android:minHeight="?attr/actionBarSize">
+
+            <androidx.constraintlayout.helper.widget.Flow
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintBottom_toBottomOf="parent"
+                android:layout_marginTop="12dp"
+                android:layout_marginBottom="12dp"
+                app:flow_horizontalGap="12dp"
```

Comment:
Can we add `app:flow_verticalGap="8dp"` and `android:layout_marginEnd="12dp"` to avoid the button reaches to the end of the toolbar? and the vertical gap gives the space between title and the button too.